### ISSUE TITLE
improve cache heards of the UI's assets

### ISSUE THREAD

**Issue by jcarres-mdsol** [openzipkin/zipkin#988]
Zipkin UI renders very fast once all the assets are retrieved.

app.min.js has `max-age=600, must-revalidate` cache headers and it's being sent every time, that's the main source of waiting on the UI for me at least.
I think adding better cache headers will make wonders for UX
@eirslett  probably interested

**Comment by eirslett** [comment]
It was added here: https://github.com/openzipkin/zipkin/commit/ac33a58
I'm pretty sure that should be the correct cache headers...

**Comment by codefromthecrypt** [comment]
oh snap! back at ya jordi! :P

**Comment by eirslett** [comment]
Works in my browser:

![from-cache](https://cloud.githubusercontent.com/assets/1872593/13143529/d3a4aa12-d644-11e5-9b15-672ee036400f.png)

**Comment by yurishkuro** [comment]
I'm also hitting cache, except for the first time

![image](https://cloud.githubusercontent.com/assets/3523016/13148030/91681afa-d629-11e5-86bf-74c7e76bd21c.png)

It's probably that first time that I've been noticing, since the cache is only for 10min. If the list was permanently stored in the browser in local storage, the first load after 10min would still be unnoticeable. Is it possible to refresh the list in the background after first displaying from local storage? Or would it completely disrupt the experience if the user has already started interacting with the dropdown?

**Comment by eirslett** [comment]
This issue is about caching assets/css/js specifically, there's another issue regarding caching of API calls. I suggest you repost the comment there!
Regarding assets caching, I'll close this issue, since we already serve cache headers.

**Comment by jcarres-mdsol** [comment]
<img width="487" alt="screen shot 2016-02-19 at 8 42 48 am" src="https://cloud.githubusercontent.com/assets/1803073/13161981/da94f214-d6e4-11e5-963a-361b64e6bede.png">

**Comment by jcarres-mdsol** [comment]
<img width="1365" alt="screen shot 2016-02-19 at 8 43 07 am" src="https://cloud.githubusercontent.com/assets/1803073/13161984/e2854e38-d6e4-11e5-842b-f6ad2f8e4a93.png">

**Comment by jcarres-mdsol** [comment]
Even after refreshing a couple of times, it does not seem to work for me :no_good: 
I've tried both chrome and firefox, just in case

**Comment by eirslett** [comment]
When you hit the refresh button, it clears the cache... try opening the UI, go to the dependencies page, then back to the index page, is it loaded from cache then?

**Comment by jcarres-mdsol** [comment]
0_0! you are right, it's kind of embarrasing not to realize till now :hear_no_evil:
